.github/workflows/push.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       id-token: write
     with:
       plugin-version-suffix: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.sha || '' }}
-      run-playwright-with-grafana-dependency: ">=11.6 <12.1"
+      run-playwright-with-grafana-dependency: ">=12.0 <13.0"
```

Comment:
there a difference in px between 11.6 and 12.0+ so the snapshots were failing. So skipped the 11.6 and decided to test versions >=12.0